core/src/main/scala/sttp/client3/HttpVersion.scala

```diff
@@ -0,0 +1,8 @@
+package sttp.client3
+
+// FIXME: Move to sttp-model ?
+sealed trait HttpVersion
+case object Empty extends HttpVersion
```

Comment:
yeah moving to model once the backends work is done would be a good idea

also I would rename `Empty` to `Default` and move the enumeration values inside a `HttpVersion` object

unfortunately there are no tags on `Response` so I guess we won't be able to add a `.protocol` method there; something for the next major version :)